llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -753,9 +753,7 @@ define void @foo(ptr %arg0, ptr %arg1) {
   auto *VLd = cast<sandboxir::LoadInst>(&*It++);
   auto *Ret = cast<sandboxir::ReturnInst>(&*It++);
 
-  // Check isVolatile()
```

Comment:
Let's keep this because the reset of the checks have a comment.

Reply:
fixed.